### PR TITLE
Added quiet mode so that we can use this more easily with raspbian-ua…

### PR DESCRIPTION
**Pull request by gbkersey**
…-netinst

Just added variable for to quiet down curl when we use rpi-update in other scripts.  No other changes.

### CODE REVIEW

File: rpi-update [hunk 1]
```diff
@@ -42,11 +44,15 @@ FW_REV=${1:-""}
 FW_REVFILE="${FW_PATH}/.firmware_revision"
 [ "${RPI_UPDATE_UNSUPPORTED}" -ne 0 ] && echo -e "You appear to be trying to update firmware on an incompatible distribution. To force update, run the following:\nsudo -E RPI_UPDATE_UNSUPPORTED=0 rpi-update" && exit 1
 
+if [[ ${QUIET_MODE} -eq 1 ]]; then
+    CURL_COMMAND="curl -s"
```
Comment (lurch) [inline rpi-update]: Presumably this should be `CURL_COMMAND="${CURL_COMMAND} -s"` to preserve any user-supplied `$CURL_COMMAND` from line 37 ?

File: README.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ To just get a list of commits contained in rpi-update since you last updated, ru
 
 This won't update your firmware
 
+#### `QUIET_MODE`
+
+Makes curl quiet.
+
+    sudo QUIET_MODE=1 rpi-update
+
+This only changes the verbosity of curl.
```
Comment (lurch) [inline README.md]: If it _only_ affects the curl quietness, maybe it should be `QUIET_CURL_MODE` instead? :wink:

**Comment by lurch** [comment]
Now that #279 has been merged, you could do this with just `sudo CURL_OPTIONS=-s rpi-update`
:smiley:

File: README.md (hunk 2 above)
Comment (gbkersey) [reply README.md]: It's been so long since I submitted this (Dec 31, 2016) that I'd forgotten all about it...  

Thanks!

**Comment by lurch** [comment]
@gbkersey Can this be closed now then?

**Comment by gbkersey** [comment]
Of course.... 
The purpose was to make raspbian-ua-netinst easier... But It's been so long, I really can't remember. 

I'm not giving you greif about it. I really appreciate what you do for raspbian. 

Thanks!

**Pull request closed without merging**